lib/utils/getoptions.js

```diff
@@ -73,4 +79,9 @@ module.exports = function cwdResolver( callOptions, cwd ) {
  * If a string: name of branch that should be created.
  *
  * @property {Boolean|undefined} [hash=undefined] Whether to use current commit hashes as an input data.
+ *
+ * @property {Object} [overrideDirectoryNames={}] A map that allows renaming directories where packages will be cloned.
+ *
+ * @property {String|Array.<String>|undefined} [packagesPrefix=[]] Prefix or prefixes which will be removed from packages' names during
```

Comment:
```suggestion
 * @property {String|Array.<String>} [packagesPrefix=[]] Prefix or prefixes which will be removed from packages' names during
```

Since it's optional, we know that it can be undefined.